### ISSUE TITLE
Wrong User class used by artisan command

### ISSUE THREAD

**Issue by CreativeBulma** [mateusjunges/laravel-acl#185]
Hi,

we can modify the User class to use in `acl.php` config file but UserPermisison command file (`Console/Commands/UserPermissions.php`) is still using the default `App\User` class. Which leads in error when running artisan command `php artisan user:permissions 1`.

```
Class 'App\User' not found

  at vendor/mateusjunges/laravel-acl/src/Console/Commands/UserPermissions.php:45
    41|     {
    42|         try {
    43|             $userParameter = $this->argument('user');
    44|             if (is_numeric($userParameter)) {
  > 45|                 $user = User::find((int) $userParameter);
    46|             } elseif (is_string($userParameter)) {
    47|                 $table = config('acl.tables.users');
    48|                 $columns = $this->verifyColumns($table);
    49| 

      +14 vendor frames 
  15  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

**Comment by mateusjunges** [comment]
Good morning @CreativeBulma, 

Thank you for your report. I'll release a minor version with this bug fix.

**Comment by mateusjunges** [comment]
@CreativeBulma fixed with version [2.4.6](https://github.com/mateusjunges/laravel-acl/tree/2.4.6)

**Comment by CreativeBulma** [comment]
thanks a lot for the reactivity.
